_posts/2021-09-07-design-of-the-koji-room-koji-muro.md

```diff
@@ -0,0 +1,461 @@
+| Miso rice koji (polishing ratio 90%)   |                      40-45 |                                  2.5 |                                41.8-64.9 |                                  460-750 |
+| Seishu rice koji (polishing ratio 75%) |                         35 |                                  2.2 |                                       30 |                                  380-440 |
+| Shoyu koji (soybean & wheat 1:1)       |                      40-45 |                                  2.9 |                                      130 |                                1800-2300 |
+
+
+<div class="callout">
+💡 Seishu (sake) rice koji generates the least amount of heat among koji types, for a simple reason: it's the least nutritious. The more you polish rice, the more you strip away lipids, proteins, and minerals which are concentrated in the outer layers. Highly polished rice is little more than a ball of starch with a bit of protein and minerals - so the koji grows slowly. By comparison - shoyu koji, which contains whole soybeans and wheat, is choc full of nutrition and generates over 4x the heat of seishu koji. 
+</div>
+
+
+With these parameters, the remainder of the process is fairly simple: all we have to do is go through four design calculations and determine, in order, the room dimensions, insulation, ventilation, and heating. 
+
+### 1. Dimensions 
+
+Let's determine the size of our table (*toko*) and koji room. To fit 300kg of steamed rice for inoculation (*tane-kiri*) and at the end of the process (*shimai shigoto*) at a depth of 5cm, we need a 13.2 square meter table, housed in a koji room measuring about **8m(L) x 5m (W) x 2.2m(H)** in order to fit clearances and other equipment (See Appendix - Step 1, 2).
+
+### 2. Insulation
+
+The koji room's insulation must be sufficient to keep the room warm AND prevent condensation on the inside surfaces of the room.  If we look at a section of the koji-room wall, overlaid with a steady-state temperature profile, it will look something like this:
+
+![](/assets/images/koji_room/Wallheattransfer.png){:height="550" .center}
+*Depiction of 1-D steady state heat transfer through a wall*
+
+Our koji room, sitting at 30°C and 70% RH, will be slightly colder near the inside wall in an area known as the **temperature boundary layer**. The temperature right up against the interior wall *must* be greater than the dew point of the air in the koji room (which is 23.9°C) or condensation will form. 
+
+This is a 1-dimensional heat transfer problem. If we pick 4-inch polyurethane with a total RSI-value of 4.24, we can demonstrate that the wall temperature will not fall below 29.3°C. (see Appendix - Step 3). This might seem like overkill - our design requirement was 23.9°C! But trust me, insulation is cheap relative to the rest of the room; and the heat retention will save costs in the long run. 
```

Comment:
Maybe a small inline description of RSI-value?

Reply:
For the reader's aid, you can hotlink to the appendix anchor points. Ex:

```
(see [Appendix - Step 3](#step-3---determine-heat-loss-through-the-room-walls-and-check-for-condensation))
```
will create a link to Step 3 on the page. 

Here are the other "ids" for your appendix:
```
step-1---determine-the-room-and-outdoor-conditions
step-2---determine-the-koji-amount-dry-weight-and-room-dimensions
step-4---determine-the-ventilation-requirements-for-heat-removal
step-5---determine-the-ventilation-requirements-for-moisture-removal
step-6---determine-the-ventilation-requirements-for-co-removal
step-7---determine-the-heating-requirements
```

Reply:
Added! I ought to learn how the whole system you use for this blog works (jekyll?) unless this is unrelated